Libraries/Components/Touchable/TouchableWithoutFeedback.js

```diff
@@ -41,6 +41,10 @@ var TouchableWithoutFeedback = React.createClass({
       React.PropTypes.arrayOf(React.PropTypes.oneOf(View.AccessibilityTraits)),
     ]),
     /**
+     * Disable all component interactions if value is true
```

Comment:
nit: if the value is true / If true, disable all interactions for this component.